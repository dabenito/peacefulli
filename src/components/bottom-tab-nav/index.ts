export { Nav } from "./bottom-tab-nav";
