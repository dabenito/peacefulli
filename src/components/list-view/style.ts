import { StyleSheet } from 'react-native';
import { Color } from '../../constants/color-constants';

export const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: Color.white,
        alignItems: 'center',
        justifyContent: 'center',
    },
    listContainer: {
        flex: 4,
        backgroundColor: Color.white,
        alignItems: 'center',
        justifyContent: 'center',
        marginTop: -75,
    },
});
