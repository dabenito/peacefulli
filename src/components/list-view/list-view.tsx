import React, { PureComponent } from "react";
import { View, Text } from "react-native";
import { styles } from "./style";

export interface ListViewProps {}

export interface ListViewState {}

export class ListView extends PureComponent<ListViewProps, ListViewState> {
  constructor(props: ListViewProps) {
    super(props);
  }

  render() {
    return (
      <View style={styles.container}>
        <Text>TODO: Create List View!</Text>
      </View>
    );
  }
}
