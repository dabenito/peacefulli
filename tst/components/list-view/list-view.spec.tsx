import React from "react";
import { render, cleanup } from "@testing-library/react-native";
import { ListView } from "../../../src/components/list-view";

describe("ListView Tests", () => {
  afterEach(cleanup);

  // TODO: update this file when implementing listView page
  it("loading listView page", async () => {
    const listView = render(<ListView />);
    expect(listView).toMatchSnapshot();
  });
});
