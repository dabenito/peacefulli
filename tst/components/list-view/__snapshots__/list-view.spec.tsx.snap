// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListView Tests loading listView page 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    TODO: Create List View!
  </Text>
</View>
`;
